service-api/module/Application/config/module.config.php

```diff
@@ -330,13 +341,13 @@
                     ],
                 ],
             ],
-            'save_alternate_address_to_case' => [
+            'update-professional-address' => [
```

Comment:
Ideally maintain the convention of underscores in route names?

Reply:
Yes of course. I've updated that now.